### DISCUSSION TITLE
Remembering split view settings in Xaml design view

### DISCUSSION THREAD

**Discussion by cuikp** [AvaloniaUI/AvaloniaVS#414] [General]
Since the default split view for Xaml design is left-right, and myself preferring almost exclusively the top/bottom split view for Xaml, I wonder if it's an easy matter to either:
1. Have Avalonia projects remember the split view settings either on the project level or axaml level, or
2. Have a project setting to set the default split view for all axaml files.?

**Comment by timunie** [comment]
We need some context here. Which IDE? Which version of the extension?

**Reply by thevortexcloud** [reply]
I know in Visual Studio it seems to remember it per file. I personally almost always disable the previewer since it does not work with how I structure some code or for some reason tries to run on non previewable files such as Resource Dictionaries. Which ends up throwing errors. But I still have to minimise it for every file when I would prefer to have it disabled by default and then manually enabled per file.

**Reply by cuikp** [reply]
Sorry, the IDE is Visual Studio 2022, Avalonia 11.0.4 (~latest version)

**Comment by stevemonaco** [comment]
The Avalonia XAML Editor in VS Options has settings for this globally. Have you tried that? I use it to change the Default Document View (Source) and it works. However, it tends to occasionally reset to defaults and I'm not sure what does it. Either a VS or AvaloniaVS extension update, probably.

![image](https://github.com/AvaloniaUI/AvaloniaVS/assets/1782158/6e5745e1-5ad5-4f0b-9947-59f744e948c9)

**Reply by cuikp** [reply]
Ah ha! There it was! What was I thinking not checking for an Avalonia Xaml Editor item in Options - Tools...  
Thanks - that seems to work.  I'll see if it occasionally resets here as well.
